### ISSUE TITLE
Consumet api

### ISSUE THREAD

**Issue by scara78** [N0-0NE-Dev/NoFasel#53]
This app is great, if you implement api from consumet.org (https://github.com/consumet/api.consumet.org), you have all the information provided by json from api:

- Poster image
- Description of the movie
- Related movies (directly from json)
- Direct HLS stream from Vidcloud or Upcoud (good quality and speed)

I have deployed their api on vercel and played for a few monts with, in my projects, its really good!

Have a nice day

**Comment by N0-0NE-Dev** [comment]
Thanks for bringing this awesome project to my attention, I think it's time for NoFasel 2!

**Comment by Leichesters** [comment]
@N0-0NE-Dev I agree that this would be a good time to rewrite the structure. How about we go with a netflix style for NoFasel 2?

**Comment by N0-0NE-Dev** [comment]
> @N0-0NE-Dev I agree that this would be a good time to rewrite the structure. How about we go with a netflix style for NoFasel 2?

I need to give consumet a closer look to know what will and won't be possible and also I am waiting for a couple of pull requests on a couple of dependencies to be merged to make things like NoFasel windows possible

**Comment by Leichesters** [comment]
Okay feel free to reach out if you need any help :D

btw, what do you mean by "NoFasel windows"?

**Comment by N0-0NE-Dev** [comment]
> btw, what do you mean by "NoFasel windows"?

A version of NoFaslel that runs on windows.

**Comment by scara78** [comment]
Can wait to see the new version with consumet..

**Comment by DarkCrypt** [comment]
Not trying to be an ass, just pointing out something from months ago.

[Sora](https://sora.is-really.cool/) was brought to your attention in comment #9 and Consumet is in that list. But that doesn't matter since you're on board. Now you have all the resources to make your app badass!

**Comment by OshekharO** [comment]
Also you may take into account this project: https://github.com/koikiss-dev/jimov_api ( Good For Anime Contents )
